modules/navier_stokes/include/physics/WCNSFVFlowPhysics.h

```diff
@@ -50,6 +57,8 @@ class WCNSFVFlowPhysics final : public NavierStokesPhysicsBase
   const std::vector<Point> & getFluxInletDirections() const { return _flux_inlet_directions; }
   /// Get the inlet flux postprocessor if using a flux inlet
   const std::vector<PostprocessorName> & getFluxInletPPs() const { return _flux_inlet_pps; }
+  /// Get the name of the linear friction coefficient
```

Comment:
Add in the doxygen that this will return something "invalid" if there's no friction

Reply:
done